### DISCUSSION TITLE
Shadow helper not working in 5.1?

### DISCUSSION THREAD

**Discussion by lowwhistler** [twbs/bootstrap#34816] [Q&A]
Not sure what's happening, but for me, Shadow is rendering the box black. Even the docs page has them rendered black in Chrome and Firefox?

<img width="1058" alt="Shadow" src="https://user-images.githubusercontent.com/3652907/130652666-ed8b9369-bd3b-4e53-8da3-eb30657802a1.png">

**Comment by adityadees** [comment]
this because of `.bg-body`
you can remove `.bg-body` or maybe change it to ex: `bg-primary`, `bg-light` etc.

**Reply by lowwhistler** [reply]
Yes, that's definitely the issue, thanks. Don't really want to replace everywhere, so I'll just sit tight and wait for the next update. Thanks for the reply
